nixos/tests/common/systemd-journal-remote/generate-certs.nix

```diff
@@ -0,0 +1,36 @@
+# Minica can provide a CA key and cert, plus a key
+# and cert for our fake CA server's Web Front End (WFE).
+{
+  pkgs ? import <nixpkgs> {},
+  minica ? pkgs.minica,
+  mkDerivation ? pkgs.stdenv.mkDerivation
+}:
```

Comment:
This file is not used anywhere, is it?

Reply:
It is there just so that we know how to regenerate the certificates present in the same directory. I followed the way it's done for ACME tests: https://github.com/NixOS/nixpkgs/tree/master/nixos/tests/common/acme/server